translations/ja.json

```diff
@@ -348,57 +348,54 @@
-	"Local proxy ignored: not passing {{.name}}={{.value}} to docker env.": "",
-	"Location of the VPNKit socket used for networking. If empty, disables Hyperkit VPNKitSock, if 'auto' uses Docker for Mac VPNKit connection, otherwise uses the specified VSock (hyperkit driver only)": "ネットワーキングに使用する VPNKit ソケットのロケーション。空の場合、Hyperkit VPNKitSock が無効になり、「auto」の場合、Mac VPNKit 接続に Docker が使用され、それ以外の場合、指定された VSock が使用されます（hyperkit ドライバのみ）",
-	"Location of the minikube iso": "minikube iso のロケーション",
-	"Locations to fetch the minikube ISO from.": "",
-	"Log into or run a command on a machine with SSH; similar to 'docker-machine ssh'.": "",
+	"Launching proxy ...": "プロキシーを起動しています...",
+	"List all available images from the local cache.": "ローカルキャッシュから利用可能な全イメージを一覧表示します。",
+	"List existing minikube nodes.": "既存の minikube ノードを一覧表示します。",
+	"List image names the addon w/ADDON_NAME used. For a list of available addons use: minikube addons list": "ADDON_NAME アドオンが使用しているイメージ名を一覧表示します。利用可能なアドオンの一覧表示は、次のコマンドを実行してください: minikube addons list",
+	"List images": "イメージを一覧表示します",
+	"List nodes.": "ノードを一覧表示します。",
+	"List of guest VSock ports that should be exposed as sockets on the host (hyperkit driver only)": "ホスト上でソケットとして公開する必要のあるゲスト VSock ポートの一覧 (hyperkit ドライバーのみ)",
+	"List of ports that should be exposed (docker and podman driver only)": "公開する必要のあるポートの一覧 (docker、podman ドライバーのみ)",
+	"Listening to 0.0.0.0 on external docker host {{.host}}. Please be advised": "外部 Docker ホスト {{.host}} 上で 0.0.0.0 をリッスンしています。ご承知おきください",
+	"Listening to {{.listenAddr}}. This is not recommended and can cause a security vulnerability. Use at your own risk": "{{.listenAddr}} をリッスンしています。これは推奨されず、セキュリティー脆弱性になる可能性があります。自己責任で使用してください",
+	"Lists all available minikube addons as well as their current statuses (enabled/disabled)": "利用可能な minikube アドオンとその現在の状態 (有効 / 無効) を一覧表示します。",
+	"Lists all minikube profiles.": "minikube プロファイルを一覧表示します。",
+	"Lists all valid default values for PROPERTY_NAME": "PROPERTY_NAME 用の有効な minikube プロファイルを一覧表示します",
+	"Lists all valid minikube profiles and detects all possible invalid profiles.": "有効な minikube プロファイルを一覧表示し、無効の可能性のあるプロファイルを全て検知します",
+	"Lists the URLs for the services in your local cluster": "ローカルクラスターのサービス用 URL を一覧表示します",
+	"Load a image into minikube": "minikube にイメージを読み込ませます",
+	"Local folders to share with Guest via NFS mounts (hyperkit driver only)": "NFS マウントを介してゲストと共有するローカルフォルダー (hyperkit ドライバーのみ)",
+	"Local proxy ignored: not passing {{.name}}={{.value}} to docker env.": "ローカルプロキシーは無視されました: docker env に {{.name}}={{.value}} は渡されません。",
+	"Location of the VPNKit socket used for networking. If empty, disables Hyperkit VPNKitSock, if 'auto' uses Docker for Mac VPNKit connection, otherwise uses the specified VSock (hyperkit driver only)": "ネットワーキングに使用する VPNKit ソケットのロケーション。空の場合、Hyperkit VPNKitSock が無効になり、'auto' の場合、Mac VPNKit 接続に Docker が使用され、それ以外の場合、指定された VSock が使用されます (hyperkit ドライバーのみ)",
```

Comment:
```suggestion
	"Location of the VPNKit socket used for networking. If empty, disables Hyperkit VPNKitSock, if 'auto' uses Docker for Mac VPNKit connection, otherwise uses the specified VSock (hyperkit driver only)": "ネットワーキングに使用する VPNKit ソケットのロケーション。空の場合、Hyperkit VPNKitSock が無効になり、'auto' の場合、Docker for Mac のVPNKit 接続が使用され、それ以外の場合、指定された VSock が使用されます (hyperkit ドライバーのみ)",
```
Docker package run on Mac is called ["Docker for Mac"](https://docs.docker.jp/docker-for-mac/index.html) usually, so here we should use "Docker for Mac" as one word, I think.

Reply:
Nice suggestion!
It is better to add blank before VPNKit.

Docker for Mac のVPNKit 接続が使用され
Docker for Mac の VPNKit 接続が使用され